server/eclipse-project/src/main/java/us/freeandfair/corla/model/CountyContestResult.java

```diff
@@ -507,18 +507,24 @@ public void updateResults() {
       sorted_totals.get(e.getValue()).add(e.getKey());
     }
     // next, get the winners and losers
-    // TODO: this probably needs work to deal properly with ties in 
-    // races where only one winner is allowed
-    // TODO: this needs to be revised once we save number of winners in addition
-    // to votes allowed
     final Iterator<Entry<Integer, List<String>>> vote_total_iterator = 
         sorted_totals.entrySet().iterator();
     Entry<Integer, List<String>> entry = null;
     int votes = 0;
-    while (vote_total_iterator.hasNext() && my_winners.size() < my_votes_allowed) {
+    while (vote_total_iterator.hasNext() && my_winners.size() < my_winners_allowed) {
       entry = vote_total_iterator.next();
       votes = entry.getKey();
-      my_winners.addAll(entry.getValue());
+      final List<String> choices = entry.getValue();
+      if (choices.size() + my_winners.size() < my_winners_allowed) {
```

Comment:
Should this be `<=`? We do want to add this loop iteration's `choices` if adding them to `winners` causes the new `winners` size to _exactly_ match `my_winners_allowed`, right?

Reply:
I’ll look into that.

Reply:
Not only should it have been, but there was a problem in the code below it too.

Reply:
Addressed in 86abdb60